deploy.rb

```diff
@@ -15,10 +15,10 @@
 
 WORK_DIR = ['tmp/repos'].freeze
 
-def update_repo(repo_dir)
+def update_repo(repo_dir, branch_name)
   Dir.chdir(repo_dir) do
     `git checkout config/deploy.rb`
-    `git checkout master 2> /dev/null && git pull`
+    `git checkout #{branch_name} 2> /dev/null && git pull`
```

Comment:
@peetucket @ndushay 

I wonder if we can simplify this and keep it out of our config, thereby making it easier to switch more projects' default branches:

```
branch_name = `git symbolic-ref refs/remotes/origin/HEAD | cut -d'/' -f4`.chomp
```

Then we can roll back all other changes in this PR

Reply:
ok w me if it works.